kaia/transaction/kaia_transaction_ws.py

```diff
@@ -1445,21 +1583,42 @@ def test_kaia_estimateGas_error_exceeds_allowance(self):
             "latest",
         ]
         _, error = Utils.call_ws(self.endpoint, method, params, self.log_path)
-        Utils.check_error(self, "GasRequiredExceedsAllowance", error)
+        Utils.check_error(self, "InsufficientFunds", error)
+
+    def test_kaia_estimateGas_error_exceeds_allowance(self):
+        method = f"{self.ns}_estimateGas"
+        address = test_data_set["account"]["1pebHolder"]["address"]
+        txGasPrice = test_data_set["unitGasPrice"]
+        txValue = hex(0)
+        params = [
+            {
+                "from": address,
+                "to": address,
+                "value": txValue,
+                "gasPrice": txGasPrice,
+            },
+            "latest",
+        ]
+        _, error = Utils.call_ws(self.endpoint, method, params, self.log_path)
+        #Utils.check_error(self, "GasRequiredExceedsAllowance", error)
+        self.assertIsNone(error) # The way "arg.Gas" is handled is different
```

Comment:
```suggestion
        self.assertIsNone(error) # "arg.Gas" is handled differently
```

Reply:
After merging https://github.com/kaiachain/kaia/pull/452 and [feat/v2.1.0](https://github.com/kaiachain/kaia/tree/feat/v2.1.0) merged into the dev branch, the assert should be removed.